.github/workflows/issues.yml

```
@@ -0,0 +1,31 @@
+---
+name: GH issues
+on:
+  workflow_call:
+    inputs:
+      ado_area_path:
+        description: 'ADO area path where ADO tickets should be created'
+        required: true
+        default: "2DegreesInvesting\\Test Team 1"
+        type: string
+      do-gh-to-ado:
+        description: 'Flag to run `gh-to-ado`'
+        required: false
+        default: true
+        type: boolean
+    secrets:
+      GH_TOKEN:
+        description: "GH personal access token"
+        required: false
+      ADO_TOKEN:
+        description: 'ADO personal acess token'
+        required: false
+jobs:
+  gh-to-ado:
+    if: ${{ inputs.do-gh-to-ado }}
+    uses: ./.github/workflows/issues-gh-to-ado.yml
+    with:
+      ado_area_path: ${{ inputs.ado_area_path}}
+    secrets:
+      GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

```suggestion
      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
https://docs.github.com/en/actions/security-guides/automatic-token-authentication